### PR TITLE
Update Collection.js

### PR DESCRIPTION
**Pull request by Strikerrr**
**Please describe the changes this PR makes and why it should be merged:**
Added a warning for docs for GuildMemberRoleStore.deleteAll()
I believe this necessary because some idiots may try it and fuck up their server, like myself.

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by appellation** [comment]
1. your html is wrong
2. this method does not apply only to roles

File: src/util/Collection.js [hunk 1]
```diff
@@ -389,6 +389,7 @@ class Collection extends Map {
 
   /**
    * Calls the `delete()` method on all items that have it.
+   * <warn>Warning: This deletes all the roles, this action is IRREVERSIBLE!
```
Comment (dragonfire535) [inline src/util/Collection.js]: </warn>

File: src/util/Collection.js [hunk 2]
```diff
@@ -389,6 +389,7 @@ class Collection extends Map {
 
   /**
    * Calls the `delete()` method on all items that have it.
+   * <warn>Warning: This deletes all the roles, this action is IRREVERSIBLE!</warn>
```
Comment (dragonfire535) [inline src/util/Collection.js]: Change `roles` to something else, maybe "all items in the Collection"?

**Comment by appellation** [comment]
I personally believe this warning is unnecessary.  The method description should be enough for anyone to understand what it does.

File: src/util/Collection.js (hunk 2 above)
Comment (Strikerrr) [reply src/util/Collection.js]: Done

**Comment by Lewdcario** [comment]
I am also -1 on this change, we already clarify that this method calls `.delete()` on its elements.

**Comment by iCrawl** [comment]
What was said above pretty much goes in line with what I think about this.

**Pull request closed without merging**